sdk/storage/azblob/appendblob/client.go

```diff
@@ -326,12 +316,6 @@ func (ab *Client) GetTags(ctx context.Context, o *blob.GetTagsOptions) (blob.Get
 	return ab.BlobClient().GetTags(ctx, o)
 }
 
-// CopyFromURL synchronously copies the data at the source URL to a block blob, with sizes up to 256 MB.
-// For more information, see https://docs.microsoft.com/en-us/rest/api/storageservices/copy-blob-from-url.
-func (ab *Client) CopyFromURL(ctx context.Context, copySource string, o *blob.CopyFromURLOptions) (blob.CopyFromURLResponse, error) {
```

Comment:
Any reason to remove this? AppendBlockFromURL is supported on AppendBlob so do we have a seperate API for that.

Reply:
AppendBlockFromURL is a separate API